### ISSUE TITLE
undefined method `io' for #<Minitest::Reporters::DelegateReporter:0x007fa26bb42368> (NoMethodError)

### ISSUE THREAD

**Issue by nickgnd** [kern/minitest-reporters#189]
Hi. 
In a Rails 5 project with Ruby 2.2.3 I have this error if I try to **benchmark** an alghoritm through Minitest. 
 ```bash
# Running:

/Users/Nico/.rvm/gems/ruby-2.2.3/gems/minitest-5.8.2/lib/minitest/benchmark.rb:22:in `run': undefined method `io' for #<Minitest::Reporters::DelegateReporter:0x007f925d08a858> (NoMethodError)
	from /Users/Nico/.rvm/gems/ruby-2.2.3/gems/minitest-5.8.2/lib/minitest.rb:155:in `block in __run'
	from /Users/Nico/.rvm/gems/ruby-2.2.3/gems/minitest-5.8.2/lib/minitest.rb:155:in `map'
	from /Users/Nico/.rvm/gems/ruby-2.2.3/gems/minitest-5.8.2/lib/minitest.rb:155:in `__run'
	from /Users/Nico/.rvm/gems/ruby-2.2.3/gems/minitest-5.8.2/lib/minitest.rb:129:in `run'
	from /Users/Nico/.rvm/gems/ruby-2.2.3/bundler/gems/rails-53ac41e48b7c/railties/lib/rails/commands/test.rb:9:in `<top (required)>'
	from /Users/Nico/.rvm/gems/ruby-2.2.3/bundler/gems/rails-53ac41e48b7c/activesupport/lib/active_support/dependencies.rb:302:in `require'
	from /Users/Nico/.rvm/gems/ruby-2.2.3/bundler/gems/rails-53ac41e48b7c/activesupport/lib/active_support/dependencies.rb:302:in `block in require'
	from /Users/Nico/.rvm/gems/ruby-2.2.3/bundler/gems/rails-53ac41e48b7c/activesupport/lib/active_support/dependencies.rb:268:in `load_dependency'
	from /Users/Nico/.rvm/gems/ruby-2.2.3/bundler/gems/rails-53ac41e48b7c/activesupport/lib/active_support/dependencies.rb:302:in `require'
	from /Users/Nico/.rvm/gems/ruby-2.2.3/bundler/gems/rails-53ac41e48b7c/railties/lib/rails/commands/commands_tasks.rb:128:in `require_command!'
	from /Users/Nico/.rvm/gems/ruby-2.2.3/bundler/gems/rails-53ac41e48b7c/railties/lib/rails/commands/commands_tasks.rb:86:in `test'
	from /Users/Nico/.rvm/gems/ruby-2.2.3/bundler/gems/rails-53ac41e48b7c/railties/lib/rails/commands/commands_tasks.rb:40:in `run_command!'
	from /Users/Nico/.rvm/gems/ruby-2.2.3/bundler/gems/rails-53ac41e48b7c/railties/lib/rails/commands.rb:18:in `<top (required)>'
	from /Users/Nico/.rvm/gems/ruby-2.2.3/bundler/gems/rails-53ac41e48b7c/activesupport/lib/active_support/dependencies.rb:302:in `require'
	from /Users/Nico/.rvm/gems/ruby-2.2.3/bundler/gems/rails-53ac41e48b7c/activesupport/lib/active_support/dependencies.rb:302:in `block in require'
	from /Users/Nico/.rvm/gems/ruby-2.2.3/bundler/gems/rails-53ac41e48b7c/activesupport/lib/active_support/dependencies.rb:268:in `load_dependency'
	from /Users/Nico/.rvm/gems/ruby-2.2.3/bundler/gems/rails-53ac41e48b7c/activesupport/lib/active_support/dependencies.rb:302:in `require'
	from /Users/Nico/Ruby_on_Rails/minitest_reporters/bin/rails:8:in `<top (required)>'
	from /Users/Nico/.rvm/gems/ruby-2.2.3/bundler/gems/rails-53ac41e48b7c/activesupport/lib/active_support/dependencies.rb:296:in `load'
	from /Users/Nico/.rvm/gems/ruby-2.2.3/bundler/gems/rails-53ac41e48b7c/activesupport/lib/active_support/dependencies.rb:296:in `block in load'
	from /Users/Nico/.rvm/gems/ruby-2.2.3/bundler/gems/rails-53ac41e48b7c/activesupport/lib/active_support/dependencies.rb:268:in `load_dependency'
	from /Users/Nico/.rvm/gems/ruby-2.2.3/bundler/gems/rails-53ac41e48b7c/activesupport/lib/active_support/dependencies.rb:296:in `load'
	from /Users/Nico/.rvm/rubies/ruby-2.2.3/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /Users/Nico/.rvm/rubies/ruby-2.2.3/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from -e:1:in `<main>'
 ```

The problem is due to ```minitest-reporters``` gem, infact, if I remove it from my Gemfile, the benchmark works fine and doesn't raise errors. (credits: https://github.com/seattlerb/minitest/issues/594)
Furthermore, it doesn't matter if I require ```minitest-reporters``` in my  ```test_helper.rb ``` file or not, the error still remains independently.

I've push a clean repo to reproduce the error: https://github.com/nickgnd/minitest-reporters-error

**Benchmark:**
 ```ruby
require 'test_helper'
require "minitest/benchmark"

class FibonacciBench < Minitest::Benchmark

  def fib n
    a, b = 0, 1
    n.times do
      a, b = b, a + b
    end
    a
  end

  def bench_fib
    assert_performance_linear 0.99 do |n|
      n.times do
        fib(1000)
      end
    end
  end

end
 ```

**test_helper** file
```ruby
ENV['RAILS_ENV'] ||= 'test'
require File.expand_path('../../config/environment', __FILE__)
require 'rails/test_help'

require "minitest/reporters"
Minitest::Reporters.use!

class ActiveSupport::TestCase
  # Setup all fixtures in test/fixtures/*.yml for all tests in alphabetical order.
  fixtures :all

  # Add more helper methods to be used by all tests here...
end

```

**Gemfile**
```ruby
source 'https://rubygems.org'

ruby '2.2.3'

gem 'rails', github: "rails/rails"
gem 'sprockets-rails', github: "rails/sprockets-rails"
gem 'sprockets', github: "rails/sprockets"
gem 'sass-rails', github: "rails/sass-rails"
gem 'arel', github: "rails/arel"
gem 'rack', github: "rack/rack"

# Use sqlite3 as the database for Active Record
gem 'sqlite3'
# Use ActiveModel has_secure_password
# gem 'bcrypt', '~> 3.1.7'

# Use Unicorn as the app server
# gem 'unicorn'

# Use Capistrano for deployment
# gem 'capistrano-rails', group: :development

# Use ActiveModelSerializers to serialize JSON responses
gem 'active_model_serializers', '~> 0.10.0.rc2'

# Use Rack CORS for handling Cross-Origin Resource Sharing (CORS), making cross-origin AJAX possible
# gem 'rack-cors'

group :development, :test do
  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
  gem 'byebug'
end

group :development do
  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
  gem 'spring'
end

# The problem
group :test do
  gem 'minitest-reporters'
end

# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
```

**Rails version**: ``` Rails 5.0.0.alpha ```

**RVM info**: 
```
ruby-2.2.3:

  system:
    uname:       "Darwin MacBook-Pro-di-Nicolo.local 14.3.0 Darwin Kernel Version 14.3.0: Mon Mar 23 11:59:05 PDT 2015; root:xnu-2782.20.48~5/RELEASE_X86_64 x86_64"
    system:      "osx/10.10/x86_64"
    bash:        "/bin/bash => GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin14)"
    zsh:         "/bin/zsh => zsh 5.0.5 (x86_64-apple-darwin14.0)"

  rvm:
    version:      "rvm 1.26.9 (latest) by Wayne E. Seguin <wayneeseguin@gmail.com>, Michal Papis <mpapis@gmail.com> [https://rvm.io/]"
    updated:      "9 months 14 days 21 hours 43 minutes 56 seconds ago"
    path:         "/Users/Nico/.rvm"

  ruby:
    interpreter:  "ruby"
    version:      "2.2.3p173"
    date:         "2015-08-18"
    platform:     "x86_64-darwin14"
    patchlevel:   "2015-08-18 revision 51636"
    full_version: "ruby 2.2.3p173 (2015-08-18 revision 51636) [x86_64-darwin14]"

  homes:
    gem:          "/Users/Nico/.rvm/gems/ruby-2.2.3"
    ruby:         "/Users/Nico/.rvm/rubies/ruby-2.2.3"

  binaries:
    ruby:         "/Users/Nico/.rvm/rubies/ruby-2.2.3/bin/ruby"
    irb:          "/Users/Nico/.rvm/rubies/ruby-2.2.3/bin/irb"
    gem:          "/Users/Nico/.rvm/rubies/ruby-2.2.3/bin/gem"
    rake:         "/Users/Nico/.rvm/rubies/ruby-2.2.3/bin/rake"

  environment:
    PATH:         "/Users/Nico/.rvm/gems/ruby-2.2.3/bin:/Users/Nico/.rvm/gems/ruby-2.2.3@global/bin:/Users/Nico/.rvm/rubies/ruby-2.2.3/bin:/Users/Nico/.rvm/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/webdriver:/opt/X11/bin:/usr/local/mysql/bin:/Library/Frameworks/Python.framework/Versions/3.4/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/webdriver:/opt/X11/bin:/usr/local/mysql/bin"
    GEM_HOME:     "/Users/Nico/.rvm/gems/ruby-2.2.3"
    GEM_PATH:     "/Users/Nico/.rvm/gems/ruby-2.2.3:/Users/Nico/.rvm/gems/ruby-2.2.3@global"
    MY_RUBY_HOME: "/Users/Nico/.rvm/rubies/ruby-2.2.3"
    IRBRC:        "/Users/Nico/.rvm/rubies/ruby-2.2.3/.irbrc"
    RUBYOPT:      ""
    gemset:       ""
```

**Comment by os97673** [comment]
Thank you for the report and the repo, I will look at the problem as soon as possible (I'm moving now thus it may take some time)  If you decide that I'm too slow feel free to ping me ;)

**Comment by os97673** [comment]
resolved by #188.  Please try 1.1.5

**Comment by nickgnd** [comment]
Works! Thanks a lot @os97673  :)
